docker/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM nvidia/cuda
+
+# Install Anaconda
+# Non interactive installation instructions can be found 
+# https://hub.docker.com/r/continuumio/anaconda/dockerfile
+ENV PATH /opt/conda/bin:$PATH
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update --fix-missing && apt-get install -y wget bzip2 ca-certificates \
+    libglib2.0-0 libxext6 libsm6 libxrender1 \
+    git mercurial subversion
+
+RUN wget --quiet https://repo.anaconda.com/archive/Anaconda2-5.3.0-Linux-x86_64.sh -O ~/anaconda.sh && \
```

Comment:
shall we do miniconda instead

Reply:
Changed to miniconda. However, the image size is dominated by the dependencies pulled by nlp-recipes. Currently the image size is ~13GB! Changing conda to mini conda brought it down by a GB
nlp-recipes         latest              8287350cd550        2 hours ago         12.9GB
nvidia/cuda         latest              9e47e9dfcb9a        7 weeks ago         2.83GB